datasets/dataset_imdb.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+IMDB dataset.
+inspired from:
+https://keras.io/examples/nlp/bidirectional_lstm_imdb/
+"""
+
+import keras
+from keras.utils import to_categorical
+from keras.preprocessing.sequence import pad_sequences
+from keras import models, layers
+from keras.datasets import imdb
+
+out_of_vocabulary = 2
+# We keep the 10000 first more frequent words in the datasets
+# Also the dimentionality of the input of the Embedding layer
+size_vocabulary = 20000
+# We only keep the 200 first words of the reviews
+maxlen = 200
+# Number of labels
+num_classes = 2
+
+# Not used here, but important another module
+input_shape = keras.Input(shape=(None,), dtype="int32")
```

Comment:
Hmmm, where and how is this used actually?

Reply:
I don't know, but an error is raised if input_shape isn't defined, like `preprocess_dataset_labels()`.
I will search where this come from

Reply:
`input_shape` is an argument of the constructor in dataset.py but It seems to stay unused during the computation.
Should we remove this feature? I don't know, maybe it be can be usefull in some cases